File: 5-kyu/rgb-to-hex-conversion/swift/solution.swift
func rgb(_ r: Int, _ g: Int, _ b: Int) -> String {
    return String(format: "%02X%02X%02X", clamp(r), clamp(g), clamp(b))
}

func clamp(_ n: Int) -> Int {
    return min(max(n, 0), 255)
}

File: 5-kyu/rgb-to-hex-conversion/python/solution.py
def clamp(n):
    return min(max(n, 0), 255)

def rgb(r, g, b):
    r, g, b = clamp(r), clamp(g), clamp(b)
    return '{0:02x}{1:02x}{2:02x}'.format(r, g, b).upper()